tools/lintapidiff.ml

```diff
@@ -0,0 +1,268 @@
+(* Detects newly added symbols that are missing "@since" annotations,
+   or removed symbols that didn't have "@deprecated" annotation before.
+
+   Handles: values, exceptions.
+   Ignores: variants, record fields, classes, module aliasing or includes, ...
+   Out of scope: changes in arity, parameters, ...
+
+   Missing attributes on undocumented identifiers in undocumented modules
+   are not reported.
+
+   Use lintapidiff.sh to run
+*)
+open Location
+open Parsetree
+
+module IdMap = Depend.StringMap
```

Comment:
I was looking at the code of `depend.ml` to learn how to use the parsetree, I'll remove it.